### ISSUE TITLE
guest author link optional?

### ISSUE THREAD

**Issue by lorig** [billoosijok/Guest-Author#4]
Is it possible for the Guest Author link to be optional? Meaning, if the field is left blank, can the author name not show up as a link?

**Comment by billoosijok** [comment]
This is fixed in 2.1.5 :)

**Comment by lorig** [comment]
It seems like this update may have broken the display of the text. No author name appears with or without a URL.

**Comment by lorig** [comment]
This seems to be broken now. When I try to add a guest author without a URL, it displays "#no-guest-link" instead. If I add a URL, it displays the URL. I tried this in a couple different themes with the same results. Someone else suggested that this is a complicated issue because of the variety of themes and how they handle the author meta. Thank you for your work on it though.

**Comment by billoosijok** [comment]
Can you send me a link where this is happening? I'm having a hard time replicating the issue. It works fine for me

**Comment by lorig** [comment]
Created a temporary post on our news site: https://news.warrington.ufl.edu/uncategorized/testing-guest-author-plugin/

**Comment by billoosijok** [comment]
@lorig Thanks! I'm looking at it right now

**Comment by billoosijok** [comment]
@lorig I think I figured out what the issue is. Can you update and see

**Comment by lorig** [comment]
The guest author name is showing now, but it is linked again. You can view the same temporary post to see.

**Comment by billoosijok** [comment]
Yeah unfortunately, I couldn't find a universal solution to remove the link markup that would work with all themes. So I decided to just use CSS to make it look like normal text. Try version 2.1.7 and let me know

**Comment by lorig** [comment]
It looks to me like 2.1.8 is working. Thanks!
